app/views/case_contacts/form/_contact_topic_notes.html.erb

```diff
@@ -0,0 +1,49 @@
+  <%= render "/shared/error_messages", resource: @case_contact %>
+  <%= form.fields_for(:contact_topic_answers) do |f| %>
+    <% topic = f.object.contact_topic %>
+    <% answer_id = topic.question.parameterize.underscore %>
+    <div class="card-style-1 pl-25 pr-25 mb-10">
+      <div class="d-flex justify-content-between <%= f.object.selected ? "mb-3" : nil %>"
+           data-action="click->icon-toggle#toggle"
+           data-bs-toggle="collapse"
+           data-bs-target="#<%= answer_id %>"
+           data-icon-toggle-target="margin"
+           data-controller="icon-toggle"
+           data-icon-toggle-icons-value='["lni-chevron-up", "lni-chevron-down"]'>
+           <h4 class="details__topics-label">
+             <%= f.label :question, "#{f.index + 1}. #{topic.question}" %> <span class="content-1">(optional)</span>
+           </h4>
+           <button class="bg-transparent border-0"
+                   type="button"
+                   id="<%= answer_id %>_button"
+                   aria-expanded="false">
+             <i class="lni lni-chevron-up" data-icon-toggle-target="icon"></i>
```

Comment:
The chevron is the same whether or not the card is open or closed. 
![Screenshot 2024-03-02 at 8 07 35 AM](https://github.com/rubyforgood/casa/assets/50247514/5b8d3d93-0f73-45d6-bd90-2dbd88bc3f50)